tests/framework/CodeInterfaceTests/RELAP5/test_relap5_code_interface_hybrid_dynamic_event_tree.xml

```diff
@@ -87,11 +93,11 @@
     </Models>
     <DataObjects>
       <PointSet name="RELAPout">
-        <Input>endtime,endtime2</Input>
+        <Input>endtime,endtime2,464:6,missionTime</Input>
         <Output>cntrlvar_802</Output>
       </PointSet>
       <PointSet name="RELAPout_hier" hierarchical="true">
-        <Input>endtime,endtime2</Input>
+        <Input>endtime,endtime2,464:6,missionTime</Input>
```

Comment:
Which one should be epistemic variables?

Reply:
- ```missionTime``` and ```464:6``` are epistemic (since defined within the ```HybridSampler``` XML block);
- ```endtime ``` and ```endtime2``` are aleatory (since part of the main ```DynamicEventTree``` sampler definition)